html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML FORMS</title>
+  </head>
+  <body>
+    <form action="https://www.lfz.com/uzairsspecialfriendlist/suscribe">
+      <h1>Coding Form!</h1>
+      <div>
+        <label>Email</label>
+          <div>
+        <input type="text" name="email"/></label>
+          </div>
```

Comment:
You can't have a closing div tag when the opening div tag is is the child of the label, You will want to fix this pattern elsewhere in your code as well.